### DISCUSSION TITLE
The docker compose is not working, it crashed my system, had to format it

### DISCUSSION THREAD

**Discussion by Inghosty** [Tiledesk/tiledesk#19] [General]
it is nice that this is an opensource software, we appreciate it, however with such branding us unabled users we would hope the official links to work, having to hurt our minds, trying to solve something for days even, and eventually crashing the system to a point that we have to format the system and loose all the data is something more of a virus trojan horse than an opensource software meant to change the way we do business, im sure most of us would be very appreciative to a point where we might want to partner up if not donate to the cause with such a product, it is referenced as a very useful tool, but the fact that you purposely place a docker compose file that is filled with errors and server crashing mechanism puts you at a totally different light, like you some rogue spy agent coming after us, all my containers gone, couldnt even ssh to my server, so all my databases gone. Please, please, we ask, if you are going to share your work, keep it hustle free, i have personally lost allot of sensitive time and data over how you delivered the docker compose, it has affected my business, effort and will. If its out there let it work, dont punish us for trying. I hope i get a response, soon as possible, if not i guess a campaign will be launched to share this message until the issue is addressed, dont hurt our servers, help us instead.

**Comment by sponzillo** [comment]
Hi,

Thanks for your feedback.

Can you please describe your problem in detail? We can't help without accurate description of the issues occurred.

Anyway if you want to use Tiledesk in a production envirnoment you should use a mission-critical cluster configuration (helm/kube etc.) documented here: https://developer.tiledesk.com/installation/running-tiledesk-with-kubernetes-helm

As reported in the Docker-compose README file:

"The Tiledesk Community Docker Compose file is **beta** and contains the latest **work-in-progress** deployment scripts and installs the latest development version of Tiledesk."

Our docker-compose file is not meant for prodution envs.

Andrea
